pom.xml

```diff
@@ -166,6 +166,22 @@
                 <artifactId>maven-release-plugin</artifactId>
                 <version>2.5.3</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>3.0.0-M4</version>
+                <configuration>
```

Comment:
I'd like to skip running these entirely on the Travis build - they take a lot of wasted time.  Could you [add an argument to do that](https://maven.apache.org/surefire/maven-failsafe-plugin/examples/skipping-tests.html) and set the travis build to use it?